tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/SecureConfigurationTest.java

```diff
@@ -177,8 +161,31 @@ private void internal_testSecureDurableSubscriber(ConnectionFactory connectionFa
       } catch (JMSSecurityException j) {
          //Expected exception
       }
+
+      Connection connection = null;
+
+      try {
+         connection = connectionFactory.createConnection();
+         Session session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+
+         try {
+            session.createTemporaryQueue();
```

Comment:
This is within testSecureDurableSubscriber test, it should be its own test case, testTemporaryQueue